collections/_article/dev-snapshot-godot-4-5-beta-1.md

```diff
@@ -0,0 +1,212 @@
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/godot-4-5-beta-1/recursive-focus.webm?1" type="video/webm">
+</video>
+
+There've also been a few improvements to `SVGTexture` in 4.5, such as the [dedicated importer](https://github.com/godotengine/godot/pull/105655). However, as it pertains to GUI, the biggest improvement would be [bruvzg](https://github.com/bruvzg)'s implementation of auto-scalable SVGs ([GH-105375](https://github.com/godotengine/godot/pull/105375)). This will allow users to explicitly oversample fonts and raster graphics, resulting a much crisper image overall.
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/godot-4-5-beta-1/svg-auto-scalable.webm?1" type="video/webm">
+</video>
+
+### Internationalization
+Internationalization has always been an extremely crucial part of the Godot Project, and its relative lack of coverage in our blogposts does not do it justice. [Haoyu Qiu](https://github.com/timothyqiu) allows us to break this bad habit of ours with a fantastic addition to the i18n workflow: editor previews! ([GH-96921](https://github.com/godotengine/godot/pull/96921)) Now there's much less guesswork necessary for how your projects will appear to the end-user in their native language, ensuring a consistent and clean style no matter the selection!
+
+Similarly, the ability to swap languages on-the-fly within the editor is now possible thanks to the efforts of [Tomasz Chabora](https://github.com/KoBeWi) and [GH-102562](https://github.com/godotengine/godot/pull/102562). Now users can preview *and* experience multiple language options in a single editor session.
+
+### Navigation
+The navigation team has been able to really spread their wings thanks to logic for 2D and 3D being handled independantly. [smix8](https://github.com/smix8) took the mantle on the initial split, while [AThousandShips](https://github.com/AThousandShips) brought the logic to the module system itself. This opened the door for improvements to performance across both navigation systems, and *dramatically* decreased the size of builds which exclusively target 2D.
+
+### Physics
+Our implementation of 3D interpolation has been completely overhauled, as the previous iteration had fundamental flaws with how it was structured in such a way that couldn't be addressed with a simple fix or patch. The solution already existed on 3.x, and has since been forward ported by [lawnjelly](https://github.com/lawnjelly) via [GH-104269](https://github.com/godotengine/godot/pull/104269). Now all logic is handled within the `SceneTree`, but in a manner which doesn't break any existing API. All projects will benefit from this improvement out-of-the-box.
```

Comment:
```suggestion
Our implementation of 3D interpolation has been completely overhauled, as the previous iteration had fundamental flaws with how it was structured in such a way that couldn't be addressed with a simple fix or patch. The solution already existed on 3.x, and has since been forward-ported by [lawnjelly](https://github.com/lawnjelly) via [GH-104269](https://github.com/godotengine/godot/pull/104269). Now all logic is handled within the `SceneTree`, but in a manner which doesn't break any existing API. All projects will benefit from this improvement out-of-the-box.
```